README.md

```diff
@@ -1 +1,133 @@
-# H
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="D:\Microverse\Module-1\H\im.jpg" alt="Abel Logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Set up Hello Microverse </b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Set up hello microverse Project <a name="about-project"></a>
+
+**[your_project__name]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+1.CSS
+2.HTML
+3.Linter
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Hello microverse</summary>
+  <ul>
+    <li><a href="https://style.css/">style.css</a></li>
+    <li><a href="https://index.html/">index.html </a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+1.Added  index.html file 
+2.Added style.css file 
+3.Added linter file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+     cooming soon 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Hello CR, the previous reviewer had told me to add every section except FAQ. So am I reduce all content that have coming soon please?

Reply:
hello @abel12-pro, good work following the requested changes. Please note that the live demo section should be deleted if the project is not deployed.  As a tip: You can also delete the **deployments** & **run tests** if your project does not have tests and is not deployed. You can add these sections in your Future features so that we know that the tests, live link,... will be added in later versions. 👍🏾

Reply:
thank you